core/server/config/env/config.production.json

```diff
@@ -8,5 +8,10 @@
     },
     "paths": {
         "contentPath": "content/"
+    },
+    "logging": {
+        "level": "info",
+        "rotation": true,
+        "transports": ["file"]
```

Comment:
Custom logging adapters don't have a final design, but from what i can see now, custom adapters can override what they want. They will inherit from the npm logging base and then it's up to them.

We are not restricted to anything, we could print all requests with errors in one file and all requests without errors in to one. With this PR, all http requests with and without errors land in one file. I will add a `TODO` to the issue as well to reconsider what's best  👍